src/app/models/form.server.model.ts

```diff
@@ -416,6 +416,9 @@ const compileFormModel = (db: Mongoose): IFormModel => {
         path: 'admin',
         select: 'email',
       })
+      if (!data) {
```

Comment:
fairly nitpicky, one could prefer return with a ternary op in such cases as a matter of style. holding less state is always better.

```typescript
return data ? { ... } : null
```